### ISSUE TITLE
Reorder Getting Started/install docs

### ISSUE THREAD

**Issue by philnichol** [aws-controllers-k8s/community#1121]
**What is the URL of the document?**

https://aws-controllers-k8s.github.io/community/docs/community/overview/

**Which section(s) is the issue in?**

**What needs fixing?**

Currently the instructions order is unintuitive, we install the controller (which would fail due to lack of permissions), then we set up IAM, and then restart the controller, I think it would be better if the IAM setup came first to avoid having to restart it.

**Additional context**

**Comment by vijtrip2** [comment]
> Currently the instructions order is unintuitive, we install the controller (which would fail due to lack of permissions), then we set up IAM, and then restart the controller, I think it would be better if the IAM setup came first to avoid having to restart it.

We chose to go this way because setting up IRSA is a more involved process for new users and we did not want them to jump into deep water straight away. Hence first install the controller, get familiar with it and then setup IAM permissions.

But I understand your thinking about this too and we have this issue https://github.com/aws-controllers-k8s/community/issues/1041 in backlog to help new users requiring minimal effort from them.
